.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ env:
   DTAS_VERSION: v1.2.2
 
 jobs:
-  veracode:
+  veracode_sca:
```

Comment:
We're missing the condition change we planned to do across all our repos to guarantee that SCA scans would also run on PRs, check https://github.com/Alfresco/alfresco-enterprise-repo/pull/1442/files as a reference.